.github/CONTRIBUTING.md

```diff
@@ -89,6 +89,7 @@ The easiest way to do this is to add a `phpunit.xml` file to the root of your WP
 ```xml
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
```

Comment:
Can we bump the version in the schema URL, to also highlight that we support PHPUnit 6.x?

Reply:
Good point. Done + also done for the `phpunit.xml.dist` file.